app/controllers/preview_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class PreviewController < ApplicationController
+  def show
+    keyword = Keyword.find(show_params[:id])
+
+    render partial: 'show', locals: {
+      keyword: keyword
+    }
+  end
+
+  private
+
+  def show_params
+    params.permit :id
```

Comment:
Lets keep consistent like we did in `keywords_controller.rb`
```suggestion
    params.permit(:id)
```